### PR TITLE
[fix bug] Need to call window.show() on Windows to display the window.

### PR DESCRIPTION
**Pull request by bjwktcsnzh**

### CODE REVIEW

**Comment by kasper9n** [comment]
So right now, clicking the tray icon doesn't reveal the window at all in Windows?

File: src-tauri/src/main.rs [hunk 1]
```diff
@@ -157,6 +157,10 @@ fn main() {
         };
         set_is_accessory_policy(false);
         std::thread::sleep(std::time::Duration::from_millis(5));
+        #[cfg(target_os = "windows")]
```
Comment (kasper9n) [inline src-tauri/src/main.rs]:
```suggestion
        #[cfg(not(target_os = "macos"))]
```

I'm guessing it might be needed on Linux as well

**Comment by bjwktcsnzh** [comment]
> So right now, clicking the tray icon doesn't reveal the window at all in Windows?

Yes on Windows, I don't know on Linux

**Comment by kasper9n** [comment]
Thanks!

**Pull request merged**